css/index.css

```diff
@@ -143,4 +142,124 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.navRow{
+    display: flex;
+    justify-content: space-between;
+    padding: 20px 50px 40px 0px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    font-weight: 600;
+    align-items: flex-end;
+    color:black;
+}
+
+.navRow a{
+    text-decoration: none;
+    color: black;
+}
+
+.navRow a:hover{
+    color: gray;
+}
+
+.top-content{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    margin: auto;
+    margin: 20px;
+}
+
+.text-container{
+    margin: auto;
+    padding: 0px 0px 0px 0px;
+}
+
```

Comment:
instead of using .boxes .box:nth-child you could have used .boxes .box .box1 and then the same but .box2, this will give you more control and better reusability